### DISCUSSION TITLE
References / Ссылки

### DISCUSSION THREAD

**Discussion by yuriy-zhilovets2** [PastVu/pastvu#648] [Ideas / Идеи и предложения]
Насколько я понимаю, удалению плохих фотографий, чтобы заменить их более лучшими, препятствуют две проблемы:

1. Образуются битые ссылки, которые могут быть запомнены в других местах
2. Теряются комментарии под фото

Предлагаю ввести объекты-алиасы, ссылающиеся на другое фото. Тогда старые ссылки можно будет заменить на такие ссылки, по которым откроется новое фото. Под новым же фото можно показывать комментарии от старого снимка, например, в отдельном табе.

**Comment by kabalin** [comment]
Так они не удаляются, а деактивируются, комментарии все сохраняются. Это все нужно решать в рамках задачи замены снимка #75 Если дубликат добавляется как "вариант" существующего снимка, тогда есть смысл сохранить переход со оригинальной ссылки (при условии, что фото было опубликовано), а если просто деактивировано как некачественный дубликат, то можно оставить как есть мне кажется. Может, разве что, если причина деактивации дубликат - то показывать ссылку на замену каким-то более наглядным образом чем через историю (это и сейчас можно уже сделать).
